examples/objectDetection/sketch.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ml5 Example
+Object Detection using COCOSSD
+This example uses a callback pattern to create the classifier
+=== */
+
+let video;
+let detector;
+let detections = [];
+
+function preload(){
+  detector = ml5.objectDetector("cocossd");
+}
+
+function setup() {
+  createCanvas(640, 480);
+
+  video = createCapture(VIDEO);
+  video.size(width, height);
+  video.hide();
+
+  detector.detectStart(video, gotDetections);
+}
+
+function gotDetections(results) {
+  detections = results;
+}
+
+function draw() {
+  image(video, 0, 0);
+
+  for (let i = 0; i < detections.length; i += 1) {
+    const detection = detections[i];
```

Comment:
```suggestion
    let detection = detections[i];
```

We're adopting p5.js style of using `let` even where `const` would be more typical JS.